index.html

```diff
@@ -1,18 +1,30 @@
 <!DOCTYPE html>
 <html>
 <head>
+<meta charset="UTF-8">
+<link rel="stylesheet" href = "newdle_style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
```

Comment:
Also do you know what this does? I on't know off the top of my head

Reply:
It's something to do with snapping the pixel dimensions to the size of the device viewing it. This is me attempting to get better at HTML/CSS whilst making my design phone and desktop friendly 😭